### PR TITLE
Updated slack/web-api packages.

### PR DESCRIPTION
**Pull request by VDubber**
Changes includes:
Upgrade slack/web-api package
Introduced check-format int test:ci & prepush
Formatted all straggler files.

Will attempt to upgrade integration-sdk-* packages once the integration-slack repo can handle v8.

### CODE REVIEW

File: src/__tests__/index.test.ts [hunk 1]
```diff
@@ -10,17 +10,18 @@ import fetchChannels from '../steps/fetch-channels';
...
-    validateInvocation,
-    getStepStartStates,
-    integrationSteps: [
-      teamStep,
-      fetchUsersStep,
-      fetchChannelMembersStep,
-      fetchChannels,
-    ],
-  };
+  const expectedInvocationConfig: IntegrationInvocationConfig<SlackIntegrationConfig> =
```
Comment (VDubber) [inline src/__tests__/index.test.ts:13]: formatting

File: src/index.ts [hunk 2]
```diff
@@ -8,14 +8,15 @@ import fetchChannelMembersStep from './steps/fetch-channel-members';
...
-  validateInvocation,
-  getStepStartStates,
-  integrationSteps: [
-    teamStep,
-    fetchUsersStep,
-    fetchChannelMembersStep,
-    fetchChannels,
-  ],
-};
+export const invocationConfig: IntegrationInvocationConfig<SlackIntegrationConfig> =
```
Comment (VDubber) [inline src/index.ts:11]: formatting

File: src/instanceConfigFields.ts [hunk 3]
```diff
@@ -1,29 +1,30 @@
...
-  },
-  botUserId: {
-    type: 'string',
-  },
-  authedUserId: {
-    type: 'string',
-  },
-};
+const instanceConfigFields: IntegrationInstanceConfigFieldMap<SlackIntegrationConfig> =
+  {
```
Comment (VDubber) [inline src/instanceConfigFields.ts:5]: formatting

File: src/steps/fetch-channel-members/__tests__/index.test.ts [hunk 4]
```diff
@@ -34,8 +34,8 @@ test('step data collection', async () => {
 
   expect(context.jobState.collectedRelationships.length).toBeGreaterThan(0);
 
-  const expectedCollectedRelationships: Relationship[] = context.jobState.collectedRelationships.map(
-    () => {
+  const expectedCollectedRelationships: Relationship[] =
+    context.jobState.collectedRelationships.map(() => {
```
Comment (VDubber) [inline src/steps/fetch-channel-members/__tests__/index.test.ts:38]: formatting

File: src/steps/fetch-channels/__tests__/index.test.ts [hunk 5]
```diff
@@ -30,8 +30,8 @@ test('step data collection', async () => {
   expect(context.jobState.collectedEntities.length).toBeGreaterThan(0);
   expect(context.jobState.collectedRelationships.length).toEqual(0);
 
-  const expectedCollectedEntities: Entity[] = context.jobState.collectedEntities.map(
-    (entity: Entity) => {
+  const expectedCollectedEntities: Entity[] =
+    context.jobState.collectedEntities.map((entity: Entity) => {
```
Comment (VDubber) [inline src/steps/fetch-channels/__tests__/index.test.ts:34]: formatting

File: src/steps/fetch-users/__tests__/index.test.ts [hunk 6]
```diff
@@ -31,8 +31,8 @@ test('step data collection', async () => {
   expect(context.jobState.collectedEntities.length).toBeGreaterThan(0);
   expect(context.jobState.collectedRelationships.length).toBeGreaterThan(0);
 
-  const expectedCollectedEntities: Entity[] = context.jobState.collectedEntities.map(
-    (entity: Entity) => {
+  const expectedCollectedEntities: Entity[] =
+    context.jobState.collectedEntities.map((entity: Entity) => {
```
Comment (VDubber) [inline src/steps/fetch-users/__tests__/index.test.ts:35]: formatting

**Review by austinkelleher — DISMISSED** [review]

**Review by austinkelleher — APPROVED** [review]

**Pull request merged**